sdk/web-pubsub/web-pubsub-express/src/cloudEventsProtocols.ts

```diff
@@ -23,10 +26,71 @@ export interface ConnectResponse {
   subprotocol?: string;
 }
 
+/**
+ * Success respones of the connect event.
+ */
+export interface MqttConnectResponse extends ConnectResponse {
+  /**
+   * The MQTT specific properties in a successful MQTT connection event response.
+   */
+  mqtt?: MqttConnectResponseProperties;
+}
+
+/**
+ * Response of a failed connect event.
+ */
+export interface ConnectEventErrorResponse {
```

Comment:
does it need `Event` in the name, or `ConnectErrorResponse` is also fine?

Reply:
`ConnectErrorResponse` is fine as well. Let me change it